features/sd-step.feature

```diff
@@ -15,30 +15,28 @@ Feature: Shared Steps
     Rules:
     - Package manager is available in every build.
     - Global SDK state is restored into the package manager installation/download location.
-    - Package manager can use the global SDK state to reduce installation/download time.
     - Method to update global SDK state.
     - Method to refer to installed packages via semver.
 
-    Background:
-        Given an existing pipeline with these images and packages with version:
-            | image          | package        | version        |
-            | golang:latest  | node           | ^4.0.0         |
-        And <image> image is used in the pipeline
-
-    Scenario: Use package via sd-step
+    Scenario Outline: Use package via sd-step
+        Given an existing pipeline with <image> image and <package> package:
```

Comment:
I'd prefer removing the colon (lines 22 and 33 in `sd-step.feature`, line 26 in `sd-step.js`) as well, but that's just me being picky.
Otherwise, looks good to me! 👍 
![](https://media1.giphy.com/media/pra4sJDyVYMY8/200w.gif?response_id=594471eec64bb345170de568)